### ISSUE TITLE
Table layout offset to right on some websites

### ISSUE THREAD

**Issue by GoogleCodeExporter** [classilla/tenfourfox#84]
```
SPECIFY VERSION YOU TESTED AGAINST: FF6.0 Win PX

STR: 
Go to http://www.atariage.com/forums/forum/12-atari-8-bit-computers/ 

Tested on TFF 6.0r PowerBook G4 1333 MHz.

The Atari forum layout is correct in TFF 5.0 and FF6 on Win XP. The mess-up 
starts with TFF 6.0b1. 

Letting TFF 6 send different user agent strings makes no difference. 

The Atari forum website does not validate on http://validator.w3.org. 

Trying to produce a reduced test case and re-saving the website as-is with 
SeaMonkey Composer produces a result that displays correctly with TFF6.
```

Original issue reported on code.google.com by `chtru...@web.de` on 18 Aug 2011 at 1:57
- Blocked on: #75
- Merged into: #75

Attachments:
- [TFF60r.png](https://storage.googleapis.com/google-code-attachments/tenfourfox/issue-84/comment-0/TFF60r.png)
- [FF6win.png](https://storage.googleapis.com/google-code-attachments/tenfourfox/issue-84/comment-0/FF6win.png)
